### PR TITLE
standardize scheduler and binder cache handlers for plugins

### PR DESCRIPTION
**Pull request by yuey002**
The main purpose of this PR is to limit the function calls for plugins, to prevent exposing non necessary core cache functions to the plugins.

1. Second half of the second step of https://github.com/kubewharf/godel-scheduler/issues/45. To standardize binder cache related invocation for plugins;
2. Do the same for scheduler cache handler too.

### CODE REVIEW

File: pkg/scheduler/framework/plugins/loadaware/estimator/node_metric_estimator.go [hunk 1]
```diff
@@ -18,13 +18,15 @@ package estimator
 
 import (
 	"fmt"
+	loadawarestore "github.com/kubewharf/godel-scheduler/pkg/scheduler/cache/commonstores/load_aware_store"
```
Comment (binacs) [inline pkg/scheduler/framework/plugins/loadaware/estimator/node_metric_estimator.go:21]: nit: move this to #L24-L27

File: pkg/scheduler/framework/preemption-plugins/preemption.go [hunk 2]
```diff
@@ -18,21 +18,23 @@ package preemptionplugins
 
 import (
 	"fmt"
+	preemptionstore "github.com/kubewharf/godel-scheduler/pkg/scheduler/cache/commonstores/preemption_store"
```
Comment (binacs) [inline pkg/scheduler/framework/preemption-plugins/preemption.go:21]: +1

**Review by binacs — COMMENTED** [review]
great work! lgtm overall just nits

File: pkg/binder/cache/interface.go [hunk 3]
```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package cache
 
 import (
+	commonstore "github.com/kubewharf/godel-scheduler/pkg/common/store"
```
Comment (binacs) [inline pkg/binder/cache/interface.go:20]: +1 here

File: pkg/binder/testing/framework_helpers.go [hunk 4]
```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package testing
 
 import (
+	"github.com/kubewharf/godel-scheduler/pkg/binder/framework/handle"
+	commonstore "github.com/kubewharf/godel-scheduler/pkg/common/store"
```
Comment (binacs) [inline pkg/binder/testing/framework_helpers.go:21]: +1

File: pkg/scheduler/framework/plugins/loadaware/estimator/node_metric_estimator.go (hunk 1 above)
Comment (yuey002) [reply pkg/scheduler/framework/plugins/loadaware/estimator/node_metric_estimator.go:21]: good catch, fixed~

File: pkg/binder/framework/plugins/defaultpreemption/pdbchecker_test.go [hunk 5]
```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package defaultpreemption
 
 import (
+	pdbstore "github.com/kubewharf/godel-scheduler/pkg/binder/cache/commonstores/pdb_store"
```
[the remark this answers is not in the document]
Comment (yuey002) [reply pkg/binder/framework/plugins/defaultpreemption/pdbchecker_test.go]: Thank you for the detailed review!! Fixed the imports :)

**Review by binacs — APPROVED** [review]

**Pull request merged**